.github/workflows/test-integration.yml

```diff
@@ -1,25 +1,31 @@
-on:
-    pull_request:
-    push:
-        branches: [master, develop, feature/gha-testintegration]
-env:
-    APP_ENV: test
-jobs:
-    run_test:
-        ##should be changed to the correct docker image
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v1
-
-            ## installing composer should be obselete due to running on a provisioned docker container soon.
-#            - name: install composer and update to latest version
-            #    - composer self-update --1
-            #    - composer install --prefer-dist
+name: test-integration
 
-            ## looking into how to change ant to composer scripts in the future.
-            - name: install packages
-            #install ant
-            - name: run scripts
-            ## ant should be changed to a composer script somehow
-                - ant
-                - composer test
+on: [ pull_request, push ]
```

Comment:
This will do the job just fine. But maybe to preserve some GHA clock-ticks you could limit the branches on which a test-integration run is triggered.

Before we had: 

```
on:
    pull_request:
    push:
        branches: [master, develop]
```

That might be slightly better from running on every push.

Reply:
I reverted it back to how it was.